deploy/crds/crd-certificaterequests.yaml

```diff
@@ -44,7 +44,7 @@ spec:
           name: Issuer
           type: string
         - jsonPath: .spec.username
-          name: Requestor
+          name: Requester
```

Comment:
This will change the output of `kubectl get certificaterequest`, I don't think this will break anyone however.